### PR TITLE
Minor changes to make conda work

### PR DESCRIPTION
**Pull request by jtangrot**
Corrected names of conda packages to make it possible to use conda profile. Also removed reference to environment.yml in nextflow.config, as that file is not present/used any more. BTW, the fact that environment.yml is missing results in a lint warning, don't know if that's relevant. The workflow seems to run as it should with  -profile test,conda

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by d4straub** [comment]
Awesome!

That shines a light on a forgotten problem: the conda packages/containers are actually not yet optimized. Your correction goes into the right direction but isn't perfect yet. The conda package [must not be pinned to a build because they can vary on different platforms](https://github.com/nf-core/modules#software-requirements).

File: modules/local/filter_stats.nf [hunk 1]
```diff
@@ -10,7 +10,7 @@ process FILTER_STATS {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
 
-    conda (params.enable_conda ? "pandas=1.1.5--py39ha9443f7_0" : null)
+    conda (params.enable_conda ? "pandas=1.1.5=py39ha9443f7_0" : null)
```
Comment (d4straub) [inline modules/local/filter_stats.nf:13]: Would you be able to test if the line:
`conda (params.enable_conda ? "pandas=1.1.5" : null)`
works as well, which would follow the guidelines that the build should not be pinned.

**Comment by d4straub** [comment]
> BTW, the fact that environment.yml is missing results in a lint warning, don't know if that's relevant.

That is fine. The linting is still optimized for DSL1 where an `environment.yml` is expected but linting is adapted to raise only a warning to make it compatible with pipelines following the DSL2 scheme.

File: modules/local/filter_stats.nf (hunk 1 above)
Comment (jtangrot) [reply modules/local/filter_stats.nf:13]: Ahh, didn't know about that. Your suggestion above seems to work!

**Review by d4straub — APPROVED** [review]
Great, looks good to me.

**Pull request merged**